lib/agenda/index.js

```diff
@@ -40,6 +40,7 @@ class Agenda extends EventEmitter {
     this._definitions = {};
     this._runningJobs = [];
     this._lockedJobs = [];
+    this._lockExpirationTimers = [];
```

Comment:
I believe an ES6 `Set` would be better here. It would simplify some code (see below). Like
`this._lockExpirationTimers.has(...)` and `this.this._lockExpirationTimers.delete(...)`.